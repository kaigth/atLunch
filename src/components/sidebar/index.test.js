import React from "react";
import { render } from '@testing-library/react';
import SideBar from '.';
import { initialize } from "@googlemaps/jest-mocks";

beforeEach(() => {
  initialize();
});

describe('SideBar', () => {
  test('matches snapshot', () => {
    const { container } = render(<SideBar />)

    expect(container).toMatchSnapshot();
  });
});
