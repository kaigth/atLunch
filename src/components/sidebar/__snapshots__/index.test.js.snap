// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar matches snapshot 1`] = `
<div>
  <section
    class="sidebar "
  >
    <div
      class="sidebar__container"
    >
      <ul
        class="sidebar__content"
      >
        <div
          class="sidebar__empty"
        >
          No Soup For You!
        </div>
      </ul>
    </div>
  </section>
</div>
`;
